### ISSUE TITLE
Bilingual Maori + English translation

### ISSUE THREAD

**Issue by Br3nda** [WhareHauora/wharehauora-server#336]
As well as the languages on #277 , we want to have (and default) to English/Maori bilingual (use of some maori nouns like whare and whanau, but otherwise english).

The rationale is we like use of Maori Te Reo language where we can as it NZ's first official, and most New Zealanders will recognise and can use these words, but not understand Maori grammar.

New immigrants and english-second-language users may find pure-strict english only UI easier to understand.

**Comment by amscraig** [comment]
We could, where we use Te Reo words, have it highlighted and those that don't understand could have a pop up that tells them what it means?

**Comment by Br3nda** [comment]
yup - but it wouldn't be straight forward when hooking into the translation gem ... it'll be "if in english, have a translation of the translation". We'd have to roll this ourselves.

**Comment by Br3nda** [comment]
better explanation -- the app is language agnostic. It looks up the strings (from a config file). if you're in english, it looks up the english config file.

We'd have to inject something to allow "whare" to be in the english config and detect that this work needs a translation to actual english.

which we can do, but it makes the task bigger, and the UI presentation more complex.
